presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -92,6 +92,10 @@ Because an ``OutOfMemoryError`` typically leaves the JVM in an
 inconsistent state, we write a heap dump, for debugging, and forcibly
 terminate the process when this occurs.
 
+Some security standards restrict the ``/tmp`` folder usage. This can prevent
```

Comment:
We should be more specific and discuss noexec on Linux. The temporary folder needs to be on a file system without noexec.